### PR TITLE
Basic REPL for helpers and modules - Closes #894

### LINKED ISSUE #894: Create a REPL to interact with codebase from console
While interacting with codebase and going to perform some certain database queries, currently we have to go through API calls. We need to have REPL for the app which helps to interact with the codebase from command shell.

### PR DESCRIPTION
**Pull request by nazarhussain**
Closes #894 

It gives ability to interact with code base in a way: 

```
> npm run console
lisk-core [lisk_dev] >
lisk-core [lisk_dev] > config.db.host
lisk-core [lisk_dev] > helpers.constants
lisk-core [lisk_dev] > logic.transactionInPool.transactionInPool(<id>)
lisk-core [lisk_dev] > modules.system.getBroadhash()
```
Its a very basic setup for REPL, if everyone interested in it, we can improve by time.

### CODE REVIEW

File: tasks/console.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+	multisignatures: '../modules/multisignatures.js',
+	node: '../modules/node.js',
+	peers: '../modules/peers.js',
+	signatures: '../modules/signatures.js',
+	system: '../modules/system.js',
+	transactions: '../modules/transactions.js',
+	transport: '../modules/transport.js'
+};
+
+async.auto({
```
Comment (MaciejBaj) [inline tasks/console.js]: Very similar thing is already done in test/node.js:initApplication. Can the function be reused?

Comment (nazarhussain) [reply tasks/console.js]: I will check

Comment (nazarhussain) [reply tasks/console.js]: Done.

**Comment by nazarhussain** [comment]
@diego-G That's a default behaviour or Nodejs. Do you want me to modify it and provide an exit command? FYI, In node you have to press Ctrl+C twice to exit.

**Comment by nazarhussain** [comment]
@diego-G Now you can exit form console by `.exit` command.

**Comment by diego-G** [comment]
Please, add `exit` as well as `crtl+c`.

**Comment by nazarhussain** [comment]
@diego-G Both added. But as per convention of the node REPL, every command starts with `.`. So you use `.exit` command.

**Pull request merged**